enterprise_access/apps/api_client/lms_client.py

```diff
@@ -144,41 +145,41 @@ def enterprise_contains_learner(self, enterprise_customer_uuid, learner_id):
 
         return result
 
-    def create_pending_enterprise_users(self, enterprise_customer_uuid, user_emails):
+    def create_pending_enterprise_user(self, enterprise_customer_uuid, user_email):
         """
-        Creates a pending enterprise user in the given ``enterprise_customer_uuid`` for each of the
-        specified ``user_emails`` provided.
+        Creates a pending enterprise user in the given enterprise customer for the provided email.
 
         Args:
-            enterprise_customer_uuid (UUID): UUID of the enterprise customer in which pending user records are created.
-            user_emails (list(str)): The emails for which pending enterprise users will be created.
+            enterprise_customer_uuid (UUID): UUID of the enterprise customer in which pending user record is created.
+            user_email (str): The email for which pending enterprise user will be created.
 
         Returns:
             A ``requests.Response`` object representing the pending-enterprise-learner endpoint response.
 
         Raises:
             ``requests.exceptions.HTTPError`` on any response with an unsuccessful status code.
         """
-        data = [
-            {
-                'enterprise_customer': str(enterprise_customer_uuid),
-                'user_email': user_email,
-            }
-            for user_email in user_emails
-        ]
+        data = {
+            'enterprise_customer': str(enterprise_customer_uuid),
+            'user_email': user_email,
+        }
```

Comment:
Another approach is to let this method accept a list of emails, and the caller can pass in only one if that's desired.

Reply:
:+1: 

At the time of writing, I was looking at the behavior of the enterprise API and noticed that a list triggers a different code path: 

https://github.com/edx/edx-enterprise/blob/2692c94ff8937d2a4cc7170ccf6f2cf4df5a44f5/enterprise/api/v1/views/pending_enterprise_customer_user.py#L45-L48

Initially I was worried that the list-based code path could eliminate some accuracy in the response codes, but now realize that the behavior is identical as long as the request contains a one-element list.